packages/stark-core/src/common/bootstrap/abstract-stark-main.ts

```diff
@@ -99,6 +99,12 @@ NNNNNNNN         NNNNNNNBBBBBBBBBBBBBBBBB   BBBBBBBBBBBBBBBBB
 We need great software developers like you! https://jobs.nbb.be
 		`);
 
+		if (this.environment.production) {
```

Comment:
Could we use ENV === "production" instead of this.environment.production ?
Because we already use ENV === "development" at line 118...

Reply:
Indeed, I'll fix that ;)